app/src/main/java/com/d101/frientree/di/RoomDBModule.kt

```diff
@@ -40,4 +42,13 @@ object RoomDBModule {
     fun provideJuiceDao(database: AppDatabase): JuiceDao {
         return database.juiceDao()
     }
+
+    private val migrationFrom1To2: Migration = object : Migration(1, 2) {
```

Comment:
🚫 **[ktlint]** <standard:multiline-expression-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>A multiline expression should start on a new line